### PR TITLE
[Doc] Update PyTorch interface documentation

### PR DESCRIPTION
**Pull request by victoriacity**

### CODE REVIEW

File: docs/lang/articles/basic/external.md [hunk 1]
```diff
@@ -135,3 +162,31 @@ for i in range(n):
...
+def copy_vector(x: ti.template(), y: ti.ext_arr()):
+    for i, j in ti.ndrange(n, m):
+        for k in ti.static(range(3)):
+            y[i, j, k] = x[i, j][k] # correct
+            # y[i][j][k] = x[i, j][k] incorrect
+            # y[i, j][k] = x[i, j][k] incorrect
+```
+Also, external arrays in a Taichi kernel are indexed using its **physical memory layout**. For PyTorch users,
+this implies that the PyTorch tensor [needs to be made contiguous](https://pytorch.org/docs/stable/generated/torch.Tensor.contiguous.html)
+before passed into a Taichi kernel:
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/basic/external.md:179]:
```suggestion
before being passed into a Taichi kernel:
```

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]
Some very minor corrections. Not trying to split hairs...

**Comment by k-ye** [comment]
Is this ready to merge?

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM!

**Comment by victoriacity** [comment]
@k-ye This PR is now ready to merge.

**Pull request merged**